src/Minsk/CodeAnalysis/Binding/Binder.cs

```diff
@@ -239,15 +239,15 @@ private BoundStatement BindIfStatement(IfStatementSyntax syntax)
         private BoundStatement BindWhileStatement(WhileStatementSyntax syntax)
         {
             var condition = BindExpression(syntax.Condition, TypeSymbol.Bool);
-            var body = BindLoopBody(syntax.Body, out var breakLabel, out var continueLabel);
-            return new BoundWhileStatement(condition, body, breakLabel, continueLabel);
+            var body = BindLoopBody(syntax.Body, out var bodyLabel, out var breakLabel, out var continueLabel);
```

Comment:
I don't think we should add the `bodyLabel` to the bound nodes. The reason that we have `breakLabel` and `continueLabel` is that we can access them during binding. The `bodyLabel` should just be generated during lowering, like it already happens with labels like `checkLabel`.

Reply:
Ok, I was trying to have more descriptive label names when seeing the lowered program. I reverted these changes.